doc/secret_rotation_general.md

```diff
@@ -0,0 +1,96 @@
+# Secret Rotation
+
+Note, this document explains general rotation of secrets.
+
+The instructions to rotate the CCDB keys specifically are in
+[a separate document](secret_rotation.md).
+
+The audience of this document are
```

Comment:
```suggestion
The audience of this document is:
```

Reply:
I am partial to the `are`, because what follows is all plural.

Reply:
`is` refers to `the audience`, therefore the only right form of the verb to be is `is` here.